newm-server/src/main/kotlin/io/newm/server/features/marketplace/repo/MarketplaceRepositoryImpl.kt

```diff
@@ -141,6 +130,114 @@ internal class MarketplaceRepositoryImpl(
         }
     }
 
+    override suspend fun generateSaleAmount(request: SaleAmountRequest): SaleAmountResponse {
+        log.debug { "generateCreateSaleAmount: $request" }
```

Comment:
fix debug message to match function name.

Reply:
Done